### PR TITLE
Add Healx as contributing institution

### PR DESCRIPTION
**Pull request by pinin4fjords**

### CODE REVIEW

**Review by maxulysse — APPROVED** [review]

**Comment by pinin4fjords** [comment]
Thanks @maxulysse - just waiting for some internal sign-off before I merge, hence the draft

File: nf-core-contributors.yaml
Comment (qugu) [inline nf-core-contributors.yaml:932]: There's a typo in there

Comment (pinin4fjords) [reply nf-core-contributors.yaml:932]: oops- thanks!

**Pull request merged**
